pom.xml

```diff
@@ -354,9 +354,6 @@ THE SOFTWARE.
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
-          <configuration>
-            <additionalparam>-Xdoclint:none</additionalparam>
-          </configuration>
```

Comment:
No longer necessary: Javadoc execution should now succeed (with >100 `warning`s but no `error`s).